mpas_analysis/ocean/nino34_timeseries.py

```diff
@@ -0,0 +1,209 @@
+                                timeVariableName='Time',
+                                variableList=varList,
+                                variableMap=variableMap,
+                                startDate=startDate,
+                                endDate=endDate)
+
+    SSTregions = ds.avgSurfaceTemperature
+    print '  Compute NINO3.4 index...'
+    nino34 = compute_nino34_index(SSTregions[:, regionIndex], config)
+
+    print ' Computing NINO3.4 power spectra...'
+#   NOTE: first 2 and last two values ommitted due to rolling mean
+    f, spectra, conf99, conf95, redNoise = compute_nino34_spectra(nino34, config)
+
+    start = days_to_datetime(np.amin(ds.Time.min()), calendar=calendar)
+    end = days_to_datetime(np.amax(ds.Time.max()), calendar=calendar)
+
+#   Convert frequencies to period in years
+    f = 1.0 / (1E-9 + f*86400.*365)
+    print ' Plot NINO3.4 index and spectra...'
+    xLabel = 'Time [years]'
+    yLabel = '[$^\circ$ C]'
+
+    figureName = '{}/NINO34_{}.png'.format(plotsDirectory, mainRunName)
+    nino34_timeseries_plot(config, nino34, 'NINO 3.4 Index',
+    			   xLabel, yLabel, figureName, linewidths=3, calendar=calendar)
+
+    xLabel = 'Period (years)'
+    yLabel = 'Power Spectrum'
+    figureName = '{}/NINO34_spectra_{}.png'.format(plotsDirectory, mainRunName)
+    nino34_spectra_plot(config, f, spectra, conf95, conf99, redNoise,
+			'NINO3.4 power spectrum', xLabel,
+			yLabel, figureName, linewidths=3)
+
+ 
+def compute_nino34_index(regionSST, config):
+    """
+    Computes nino34 index time series.  It follow the standard nino34 algorithm, i.e.,
+
+    1) Compute monthly average SST in the region
+    2) Computes anomalous SST
+    3) Performs a 5 point running mean over the anomalies
+
+    This routine requires regionSST to be the SSTs in the nino3.4 region ONLY.
+    It is defined as lat > -5S and lat < 5N and lon > 190E and lon < 240E.
+
+    Further, regionSST must be a dataArray
+
+    Author: Luke Van Roekel
+    Last Modified: 03/20/2017
+    """
+
+    inDirectory = config.get('input', 'baseDirectory')
+    namelistFileName = config.get('input', 'oceanNamelistFileName')
+    namelist = NameList(namelistFileName, path=inDirectory)
+    calendar = namelist.get('config_calendar_type')
+
+    # Compute monthly average climatology of SST
+    monthlyClimatology = climatology.compute_monthly_climatology(regionSST, calendar)
+
+    anomalySST = regionSST.groupby('month') - monthlyClimatology
+
+    return _running_mean(anomalySST.to_pandas())
+
+
+def compute_nino34_spectra(nino34Index, config):
+    """
+    Computes power spectra of Nino34 index.
+
+    nino34Index is the NINO index computed by compute_nino34_index
+
+    The algorithm follows the NCL cvdp package see
+    http://www.cesm.ucar.edu/working_groups/CVC/cvdp/code.html
+
+    NOTE: this routine requires the signal and stats module from scipy
+
+    Author: Luke Van Roekel
```

Comment:
@vanroekel, could you state clearly what the function returns (preferably following the convention of other newer docstrings):
```
Returns
-------
f : numpy.array
    blah blah

pxxSmooth : numpy.array
    blah blah
```